recipes/domcmc/activate.sh

```diff
@@ -0,0 +1,21 @@
+
+# make sure that a profile is loaded i.e. you are in ECCC's environment
```

Comment:
What is this script doing? It looks like it loads a dependency from some hidden place?